tests/test_preprocess_artifacts.py

```diff
@@ -2,8 +2,8 @@
 
 import inspect
 import os
-import pathlib
 from typing import Generator
+from py._path.local import LocalPath
```

Comment:
Can you please comment this decision? Is this because tmpdir creates py._path.local.LocalPath objects, or for some other reason?